classrooms/views.py

```
@@ -2,2 +1,107 @@
+    context = {
+        "classrooms": classrooms,
+        "enrolled_classes": enrolled_classes,
+        'teacher_classes': teacher_classes,
+        "role": role,
+    }
+    return render(request, "classrooms/classroom_list.html", context)
+
+
+@login_required
+@require_http_methods(["GET", "POST"])
+def handle_classroom_form(request: HttpRequest, slug: str = None) -> HttpResponse:
```

Ruff (0.8.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)